src/base/Cellar.sol

```diff
@@ -938,7 +939,7 @@ contract Cellar is ERC4626, Ownable, Multicall {
             uint256 balanceThisAccrual = positionBalances[i];
 
             // Measure yield earned against this position's high watermark.
-            int256 yield = balanceThisAccrual.toInt256() - positionData.highWatermark;
+            int256 yield = balanceThisAccrual.toInt256() - positionData.highWatermark; // highWatermark is implicitly converted to an int256
```

Comment:
Can delete this comment too. For future code comments though make sure to capitalize and use periods.